src/qtbase/QSizeF.js

```diff
@@ -0,0 +1,7 @@
+class QSizeF extends QObject {
```

Comment:
No, should be good. Btw, that's not the «real» class inheritance, it's the same prototypes with a syntax sugar =).